From 9d1be9a8461f4c42d6aaa24c2cb1522649509d31 Mon Sep 17 00:00:00 2001
From: gabrielgagz <g.a.gomez@gmail.com>
Date: Sun, 3 Feb 2019 19:30:33 -0300
Subject: [PATCH 26/35] keylayout: add missing buttons to Razer Serval

kl included in sources, lacks of SELECT and START buttons. Also, rename "mode" to "1" according to vendor stock file.

Change-Id: Id2a91dff0b9e69335f31168987a3d273a83667b3
---
 data/keyboards/Vendor_1532_Product_0900.kl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/data/keyboards/Vendor_1532_Product_0900.kl b/data/keyboards/Vendor_1532_Product_0900.kl
index c2fc1b4f936..735197c4743 100644
--- a/data/keyboards/Vendor_1532_Product_0900.kl
+++ b/data/keyboards/Vendor_1532_Product_0900.kl
@@ -20,7 +20,9 @@ key 307 BUTTON_X
 key 308 BUTTON_Y
 key 310 BUTTON_L1
 key 311 BUTTON_R1
-key 316 BUTTON_MODE
+key 314 BUTTON_SELECT
+key 315 BUTTON_START
+key 316 BUTTON_1
 key 317 BUTTON_THUMBL
 key 318 BUTTON_THUMBR
 
-- 
2.17.1

